multi-node-k8s.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+
+# enable additional sources for yum
+# (epel for ansible and golang)
+yum -y install epel-release
+
+# Install additional packages
+yum -y install \
+	qemu-kvm \
+	qemu-kvm-tools \
+	qemu-img \
+	libvirt \
+	ansible \
+	golang
+
+# install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/"${curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt}"/bin/linux/amd64/kubectl
```

Comment:
for executing a command and using its output, use `$(curl ...)` and not `${curl ...}`

Reply:
One of the recent jobs failed due to this:
```
./multi-node-k8s.sh: line 20: ${curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt}: bad substitution
+ mv kubectl /usr/bin/kubectl
mv: cannot stat ���kubectl���: No such file or directory
+ chmod +x /usr/bin/kubectl
chmod: cannot access ���/usr/bin/kubectl���: No such file or directory
```